ospo-book/content/en/06-chapter.md

```diff
@@ -1,7 +1,103 @@
+
+### Training and Education as Funding Avenues
+`🏛 Social-Gov-Oriented`
+
+The potential for integrating OSS training within government contracts is seen as a means to provide both funding for OSS organizations and valuable training for employees. This represents an approach to leveraging existing procurement processes to support OSS sustainability.
+
+
+## OSPOs for Container Capabilities
+
+Content has been curated from the [OSPOlogyLive Apeldoorn Roundtable Discussion](https://github.com/todogroup/ospology/discussions/438#discussioncomment-8620412) and based on the adoption of open source software (OSS) for container orchestration, particularly Kubernetes, and its implications for IT strategy in organizations.
+
+### OSPOs working with Container Hosting Infrastructure Teams
+
+Teams dedicated to this infrastructure, focusing on aspects such as image repositories and platform enablement, collaborate closely with OSPOs to support container infrastructure and capabilities. Initiatives like the Cloud Platform Engineering Team (CPET) has been highlighted for their efforts in publishing self-built operators, such as the Platform as a Service (PaaS) Operator, which automates tasks like initiating namespaces or projects on Kubernetes platforms.
+
+#### Commercial vs. Open Source for Kubernetes Deployment
+
+A central point of debate was the choice between adhering to premium advisors' recommendations for using commercial services for Kubernetes deployment versus exploring open source alternatives. Given the wide array of Kubernetes distributions, this decision involves complex considerations. The discussion emphasized the importance of finding a middle ground that appreciates the advantages of OSS while also recognizing the ethical imperative to contribute to its sustainability.
+
+### Choosing Additional OSS Capabilities
+`🏛 Social-Gov-Oriented`
+
+The criteria for integrating additional OSS capabilities into commercial Kubernetes platforms include the popularity of the OSS, the contributor community's size, vendor support, adherence to standards such as OpenTelemetry, and alignment with the Cloud Native Computing Foundation (CNCF) ecosystem. Concerns are raised about copyright control and the potential pitfalls of centralized code ownership.
+
+### Maturity and Reliability of OSS for Enterprise Use
+`📈 Business-Oriented`
+
+Ongoing discussion on the maturity and reliability of OSS for meeting enterprise needs for speed, robustness, and reliability. It has been acknowledged that while commercial vendors might offer support, success is not guaranteed. The necessity for organizations to have skilled teams capable of navigating the complexities of container orchestration and Kubernetes to maintain operational resilience was stressed.
+
+
+## Sustainable IT
```

Comment:
As open source project sustainability is also a topic we should define that this is environmental sustainability. 
```suggestion
## Environmentally sustainable software
```

Reply:
Is Sustainable IT a term that has wide use?  I don't think it means the same as Environmentally sustainable software as it may include other forms of sustainability, including project sustainability, open source ecosystem sustainability, and perhaps geopolitical or similar?

Reply:
@alice-sowerby correct. The roundtable where this topic was discussed was beyond software (probably that's why they call it IT) 

This is the description that is on the ospologylive apeldoorn landing page, to get some context:

*Exploring the integration of sustainable software engineering principles into open source projects, focusing on enhancing environmental impact, energy efficiency, and long-term viability. This roundtable will discuss advancing sustainability in open source development, with an emphasis on creating software that is both energy-efficient and supports sustainable community practices. We will also evaluate the role of these sustainable practices in improving the effectiveness and lifespan of open source projects, particularly within the public sector*